gfx/include/gfx/gfx_constants.h

```diff
@@ -6,10 +6,10 @@
 
 namespace antioch::gfx {
 
-constexpr uint32_t MAX_PRIMS_PER_DRAW = 50;
-constexpr uint32_t MAX_DRAWS_PER_BUFFER = 1;
+constexpr uint32_t MAX_PRIMS_PER_DRAW = 64;
```

Comment:
what's a prim

Reply:
Primitive shape, basically the objects that get drawn